coalib/collecting/Importers.py

```diff
@@ -113,20 +113,16 @@ def _iimport_objects(file_paths, names, types, supers, attributes, local):
     :raises Exception: Any exception that is thrown in module code or an
                        ImportError if paths are erroneous.
     """
-    if (file_paths == [] or
-            (names == [] and
-             types == [] and
-             supers == [] and
-             attributes == [])):
-        raise StopIteration
+    if not (file_paths and (names or types or supers or attributes)):
```

Comment:
this has breakage potential for corner cases and it's easier to understand in the original version IMO. UNack from me.